src/entities/monitor-system.entity.ts

```diff
@@ -70,7 +81,7 @@ export class MonitorSystem extends BaseEntity {
 
   @OneToMany(
     () => DerivedHrlyValue,
-    c => c.system,
+    c => c.monitorSystem,
```

Comment:
There's a build error saying it doesn't exist on DerivedHrlyValue. I would double check that file